apps/showcase/src/assets/trainings/sdk/steps/typescript-sdk/exercise/app.component.html

```diff
@@ -0,0 +1,21 @@
+<table class="table">
+  <caption align="top">
+    First 10 pets of status <b>available</b> from the Swagger Petstore
+  </caption>
+  <thead>
+  <tr>
+    <th scope="col">#</th>
+    <th scope="col">Name</th>
+    <th scope="col">Status</th>
+  </tr>
+  </thead>
+  <tbody>
+    @for (pet of pets(); track pet.id; let index = $index) {
+      <tr>
+        <td>{{index + 1}}</td>
+        <td><!-- Name of pet --></td>
+        <td><!-- Status of pet --></td>
```

Comment:
I think the exercise should be validated as soon as the developer update the signal with the values retrieved from the API
```suggestion
        <td>{{ pet.name }}</td>
        <td>{{ pet.status }}</td>
```